mailpoet/assets/js/src/automation/integrations/mailpoet/steps/send-email/edit/sender-domain-notice.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState, useMemo, useEffect } from 'react';
+import { dispatch } from '@wordpress/data';
+import { extractEmailDomain } from 'common/functions';
+import { SenderEmailAddressWarning } from '../../../../../../common/sender-email-address-warning';
+import { MailPoet } from '../../../../../../mailpoet';
+import { getContext } from '../../../context';
+import { storeName } from '../../../../../editor/store';
+
+type SenderDomainInlineNoticeProps = {
+  email: string;
+};
+
+/**
+ * SenderDomainStatusNotice is a wrapper for SenderEmailAddressWarning to use with Automations.
+ * It populates the props (mostly) with data from the context, instead of the Window object.
+ * It still uses the window object for values that are present in all MailPoet pages
+ * like mailpoet_mss_active or subscribers_count further down the tree.
+ *
+ * It updates the context when the user authorizes the email or domain
+ * so that other email steps in the automation have an updated configuration.
+ *
+ * @param email - FROM address email
+ * @returns JSX.Element
+ */
+function SenderDomainStatusNotice({
+  email,
+}: SenderDomainInlineNoticeProps): JSX.Element {
+  const [senderDomainsConfig, setSendersDomainConfig] = useState(
```

Comment:
I get a blank screen with an API key from the creator plan. As far as I understand, that is because senderDomainsConfig can be `null` when the key is not able to send via MSS or another sending method is used. I pushed dc8ec6f47 to account for that configuration.

Reply:
Thanks for the fix